[flaky] test/e2e.TestAutoscaleUpCountPods

### Auto-generated issue tracking flakiness of test
* **Test name**: test/e2e.TestAutoscaleUpCountPods
* **Repository name**: serving

<!-------------End of issue body, Please don't edit below this line------------->
<!--[DONT_MODIFY_TEST_IDENTIFIER]'test/e2e.TestAutoscaleUpCountPods' in repo 'serving'[DONT_MODIFY_TEST_IDENTIFIER]-->

A very interesting failure here:
```
  stream.go:221: D 12:56:12.042 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Updating Endpoints: ready backends: 0, not-ready backends: 3
    stream.go:221: D 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Done probing, got 3 healthy pods
    stream.go:221: D 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Revision state
    stream.go:221: D 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Not Probing on timer
    stream.go:221: D 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Handling update
    stream.go:221: I 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Updating Revision Throttler with: clusterIP = <nil>, trackers = 3, backends = 3
    stream.go:221: D 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Trackers 0/2: assignment: [10.68.0.37:8012 10.68.10.18:8012 10.68.3.37:8012]
    stream.go:221: I 12:56:12.052 activator-76558b5ccc-hbrdc [activator] [serving-tests/autoscale-up-down-up-qbytbjgk-00001] Set capacity to 2147483647 (backends: 3, index: 0/2)
=== CONT  TestAutoscaleUpDownUp
    autoscale.go:352: got 0 replicas, expected between [1, 4] replicas for revision autoscale-up-down-up-qbytbjgk-00001
    autoscale.go:143: Stopping generateTraffic
    autoscale_test.go:45: Error:  final scale didn't fulfill constraints: got 0 replicas, expected between [1, 4] replicas for revision autoscale-up-down-up-qbytbjgk-00001
    stream.go:221: D 12:56:12.885 controller-7b76d977d-c7w9b [service-controller] [serving-tests/autoscale-up-down-up-qbytbjgk] Adding to queue serving-tests/autoscale-up-down-up-qbytbjgk (depth: 1)
```

Now, if we look at time stamps 125608 KPA sees the first time `want=3`
But it gets `got=1` only at 125614 (beyond the test log, but available if you get the whole 2gigs of server logs). And we can see that it timed out at 125612.
We actually get `got=3` at 125618, whole 6 seconds after the test failed.
Probably need to relax the time we expect the the k8s machinery to mark those ready and propagate (note that activator marked all 3 as reachable at 125609.

I am going to relax the restraints on the test a bit.

--- a/test/e2e/autoscale.go
+++ b/test/e2e/autoscale.go
@@ -49,6 +49,7 @@ const (
 	// Concurrency must be high enough to avoid the problems with sampling
 	// but not high enough to generate scheduling problems.
 	containerConcurrency = 6
+	rpsTarget            = 10
 	targetUtilization    = 0.7
 	successRateSLO       = 0.999
 	autoscaleSleep       = 500

--- a/test/e2e/autoscale_test.go
+++ b/test/e2e/autoscale_test.go
@@ -69,7 +69,7 @@ func runAutoscaleUpCountPods(t *testing.T, class, metric string) {
 
 	ctx.t.Log("The autoscaler spins up additional replicas when traffic increases.")
 	// Note: without the warm-up / gradual increase of load the test is
-	// receving 503 responses (overload) from the envoy.
+	// receiving 503 responses (overload) from the envoy.
 
 	// Increase workload for 2 replicas for 90s. It takes longer on a weak
 	// boskos cluster to propagate the state. See #10218.

--- a/test/e2e/autoscale_test.go
+++ b/test/e2e/autoscale_test.go
@@ -74,7 +74,7 @@ func runAutoscaleUpCountPods(t *testing.T, class, metric string) {
 	// Increase workload for 2 replicas for 90s. It takes longer on a weak
 	// boskos cluster to propagate the state. See #10218.
 	// Assert the number of expected replicas is between n-1 and n+1, where n is the # of desired replicas for 60s.
-	// Assert the number of expected replicas is n and n+1 at the end of 60s, where n is the # of desired replicas.
+	// Assert the number of expected replicas is n and n+1 at the end of 90s, where n is the # of desired replicas.
 	AssertAutoscaleUpToNumPods(ctx, 1, 2, time.After(90*time.Second), true /* quick */)
 	// Increase workload scale to 3 replicas, assert between [n-1, n+1] during scale up, assert between [n, n+1] after scaleup.
 	AssertAutoscaleUpToNumPods(ctx, 2, 3, time.After(90*time.Second), true /* quick */)

--- a/test/e2e/autoscale_test.go
+++ b/test/e2e/autoscale_test.go
@@ -62,22 +62,24 @@ func TestRPSBasedAutoscaleUpCountPods(t *testing.T) {
 func runAutoscaleUpCountPods(t *testing.T, class, metric string) {
 	target := containerConcurrency
 	if metric == autoscaling.RPS {
-		target = 10
+		target = rpsTarget
 	}
 
 	ctx := SetupSvc(t, class, metric, target, targetUtilization)
 
 	ctx.t.Log("The autoscaler spins up additional replicas when traffic increases.")
-	// note: without the warm-up / gradual increase of load the test is retrieving a 503 (overload) from the envoy
+	// Note: without the warm-up / gradual increase of load the test is
+	// receving 503 responses (overload) from the envoy.
 
-	// Increase workload for 2 replicas for 60s
+	// Increase workload for 2 replicas for 90s. It takes longer on a weak
+	// boskos cluster to propagate the state. See #10218.
 	// Assert the number of expected replicas is between n-1 and n+1, where n is the # of desired replicas for 60s.
 	// Assert the number of expected replicas is n and n+1 at the end of 60s, where n is the # of desired replicas.
-	AssertAutoscaleUpToNumPods(ctx, 1, 2, time.After(60*time.Second), true /* quick */)
+	AssertAutoscaleUpToNumPods(ctx, 1, 2, time.After(90*time.Second), true /* quick */)
 	// Increase workload scale to 3 replicas, assert between [n-1, n+1] during scale up, assert between [n, n+1] after scaleup.
-	AssertAutoscaleUpToNumPods(ctx, 2, 3, time.After(60*time.Second), true /* quick */)
+	AssertAutoscaleUpToNumPods(ctx, 2, 3, time.After(90*time.Second), true /* quick */)
 	// Increase workload scale to 4 replicas, assert between [n-1, n+1] during scale up, assert between [n, n+1] after scaleup.
-	AssertAutoscaleUpToNumPods(ctx, 3, 4, time.After(60*time.Second), true /* quick */)
+	AssertAutoscaleUpToNumPods(ctx, 3, 4, time.After(90*time.Second), true /* quick */)
 }
 
 func TestAutoscaleSustaining(t *testing.T) {

